src/main/jte/bookings/single.jte

```diff
@@ -20,12 +20,21 @@
 <div class="absolute top-5 left-5">
     !{var bookingShareType = booking.getShareRoomType();}
     @if(bookingShareType == ShareRoomType.YES)
-        <i class="fa-solid fa-users"></i>
+        <label class="label" aria-label="${context.localize("bookings.collaboration.share_room")}"
+               title="${context.localize("bookings.collaboration.share_room")}">
+            <i class="fa-solid fa-users"></i>
+        </label>
     @elseif(bookingShareType == ShareRoomType.NO)
-        <i class="fa-solid fa-users-slash"></i>
+        <label class="label" aria-label="${context.localize("bookings.collaboration.no_share_room")}"
+               title="${context.localize("bookings.collaboration.no_share_room")}">
+            <i class="fa-solid fa-users-slash"></i>
+        </label>
     @else
-        <i class="fa-solid fa-question"></i>
-        <i class="fa-solid fa-users"></i>
+        <label class="label" aria-label="${context.localize("bookings.collaboration.on_request_share_room")}"
```

Comment:
This should be set either for aria-label or for title, not for both
Also, having this be based on the name `ON_REQUEST` and symmetric to the other tooltips would be nicer.